be/src/exec/parquet/column_chunk_reader.cpp

```diff
@@ -97,7 +97,7 @@ Status ColumnChunkReader::init() {
     auto compress_type = convert_compression_codec(metadata().codec);
     RETURN_IF_ERROR(get_block_compression_codec(compress_type, &_compress_codec));
 
-    RETURN_IF_ERROR(_try_load_dictionary());
+    RETURN_IF_ERROR(_try_load_dictionary(chunk_size));
```

Comment:
If the length of the dictionary is greater than the chunk_size is it possible to cause other overflows

Reply:
I think your issue don't related with this PR, should use another PR to fix your issue.